handlers.go

```diff
@@ -99,6 +99,8 @@ func deleteUser(w http.ResponseWriter, r *http.Request) {
 		log.Println("Delete User failed -", err)
 		http.Redirect(w, r, "/settings", http.StatusSeeOther)
 	}
+
+	// TODO(@seoyoungcho213) : Remove user id cookie
```

Comment:
What is this TODO update? is it still valid?

Reply:
Just a small change to remove the cookie from the ser's browser as a nicetude once they delete their user from our db